pymc3_hmm/distributions.py

```diff
@@ -68,25 +61,13 @@ def distribution_subset_args(dist, shape, idx, point=None):
 
     dist_param_names = dist._distr_parameters_for_repr()
 
-    if point:
-        # Try to get a concrete/NumPy value if a `point` parameter was
-        # given.
-        try:
-            idx = get_test_value(idx)
-        except TestValueError:  # pragma: no cover
-            pass
-
     res = []
     for param in dist_param_names:
 
-        # Use the (sampled) point, if present
-        if point is None or param not in point:
-            x = getattr(dist, param, None)
+        x = getattr(dist, param, None)
 
-            if x is None:
-                continue
-        else:
-            x = point[param]
+        if x is None:
+            continue
```

Comment:
I haven't been able to find a case where an element of `dist._distr_parameters_for_repr()` does not appear as an attribute of `dist` so that I can test this diff. What I've found is that typically attributes for all parameters (including those for alternative parameterizations) are set in a `Distribution` object (e.g. `mu`, `sigma`, `tau` and `sd` for `pm.Normal` and `mu`, `alpha`, `n` and `p` for `pm.NegativeBinomial`) regardless of which parameterization was used to create the object.

@brandonwillard can you think of such a case? If not perhaps we can remove this conditional.

Reply:
We can leave it uncovered; I'm sure the code works, especially given how simple it is.